docat/docat/utils.py

```diff
@@ -162,3 +166,32 @@ def should_include(name: str) -> bool:
             reverse=True,
         ),
     )
+
+
+def claim_project(project: str, db: TinyDB) -> str:
```

Comment:
Just as an optional remark, I think this function could also take an api toke as parameter.
If this token is defined it would salt the token and then use this token as the claim token.
This would then also resolve the #618 issue

Reply:
If you want, I can implement it in a second commit. Or in another PR.

Reply:
Done :+1: